src/Mgate/PubliBundle/Form/Type/RelatedDocumentType.php

```diff
@@ -22,39 +22,43 @@ public function buildForm(FormBuilderInterface $builder, array $options)
-                ));
+                'class' => 'Mgate\SuiviBundle\Entity\Etude',
+                'choice_label' => 'nom',
+                'required' => false,
+                'label' => 'Document lié à l\'étude',
+                'data' => $options['etude'],
+                'attr' => array('style' => 'min-width: 300px'),
+                'configs' => array('placeholder' => 'Sélectionnez une étude', 'allowClear' => true),
+            ));
         }
         if ($options['prospect']) {
             $builder->add('prospect', Select2EntityType::class, array(
-                    'class' => 'Mgate\PersonneBundle\Entity\Prospect',
-                    'choice_label' => 'nom',
-                    'required' => false,
-                    'label' => 'Document lié au prospect',
-                    'configs' => array('placeholder' => 'Sélectionnez un prospect', 'allowClear' => true),
-                ));
+                'class' => 'Mgate\PersonneBundle\Entity\Prospect',
+                'choice_label' => 'nom',
+                'required' => false,
+                'label' => 'Document lié au prospect',
+                'attr' => array('style' => 'min-width: 300px'),
+                'configs' => array('placeholder' => 'Sélectionnez un prospect', 'allowClear' => true),
+            ));
         }
         if ($options['formation']) {
             $builder->add('formation', Select2EntityType::class, array(
-                    'class' => 'Mgate\FormationBundle\Entity\Formation',
-                    'choice_label' => 'titre',
-                    'required' => false,
-                    'label' => 'Document lié à la formation',
-                    'configs' => array('placeholder' => 'Sélectionnez une formation', 'allowClear' => true),
-                ));
+                'class' => 'Mgate\FormationBundle\Entity\Formation',
+                'choice_label' => 'titre',
+                'required' => false,
+                'label' => 'Document lié à la formation',
+                'attr' => array('style' => 'min-width: 300px'),
+                'configs' => array('placeholder' => 'Sélectionnez une formation', 'allowClear' => true),
+            ));
         }
         if ($options['etudiant'] || $options['etude']) {
             $builder->add('membre', Select2EntityType::class, array(
-                    'label' => 'Document lié à l\'étudiant',
-                    'class' => 'Mgate\\PersonneBundle\\Entity\\Membre',
-                    'choice_label' => 'personne.prenomNom',
-                    'required' => false,
-                    'configs' => array('placeholder' => 'Sélectionnez un étudiant', 'allowClear' => true), ))
-        ;
+                'label' => 'Document lié à l\'étudiant (optionnel)',
+                'class' => 'Mgate\\PersonneBundle\\Entity\\Membre',
+                'choice_label' => 'personne.prenomNom',
+                'required' => false,
+                'attr' => array('style' => 'min-width: 300px'),
```

Comment:
CSS should be added in the HTML, not the form builder... may be too big of a change though